rtsf-at-checkout-rsp-controller-event-handler/eventhandler/process_events.go

```diff
@@ -18,61 +17,52 @@ const (
 	RFIDEventMoved    = "moved"
 	ROIActionEntered  = "ENTERED"
 	ROIActionExited   = "EXITED"
-)
 
-var loggingClient logger.LoggingClient
+	resourceName = ""
+	sourceName   = "rfid-roi-event"
+	deviceName   = "device-rfid-roi-rest"
+	profileName  = ""
```

Comment:
Can't have empty names. So we make up something since there isn't a profile for this.

```suggestion
	resourceName = "rfid-roi"
	sourceName   = "rfid-roi"
	deviceName   = "device-rfid-roi-rest"
	profileName  = "rfid-roi"
```